packages/init/.size-limit.cjs

```diff
@@ -0,0 +1,26 @@
+const pkg = require("./package.json");
+
+module.exports = [
```

Comment:
what is this for?

Reply:
Answered here: https://github.com/prismicio/slice-machine/pull/799#discussion_r1115132808